lib/services/devices/deviceService.js

```diff
@@ -99,25 +100,34 @@ function createInitialEntityHandler(deviceData, newDevice, callback) {
  */
 function createInitialEntity(deviceData, newDevice, callback) {
     var options = {
-        url: 'http://' + config.getConfig().contextBroker.host + ':' + config.getConfig().contextBroker.port +
-            '/v1/updateContext',
-        method: 'POST',
-        json: {
-            contextElements: [
-                {
-                    type: deviceData.type,
-                    isPattern: 'false',
-                    id: String(deviceData.name),
-                    attributes: []
-                }
-            ],
-            updateAction: 'APPEND'
+            url: 'http://' + config.getConfig().contextBroker.host + ':' + config.getConfig().contextBroker.port +
+                '/v1/updateContext',
```

Comment:
Usually I prefer not to extract this kind of paths to constant files, unless they are really mutable, as it reduces readability without really improving maintainability. I've moved other things, like the command and status prefixes, though, as they proved to be kind of unstable.

NTC